src/Symfony/Component/Config/Definition/Configuration.php

```diff
@@ -31,6 +31,9 @@ public function __construct(
     ) {
     }
 
+    /**
+     * @return TreeBuilder<'array'>
+     */
```

Comment:
Not sure why psalm wants this be added shouldn't it not just use interface return type here and not force redefinition?